internal/profile/profile.go

```diff
@@ -305,18 +327,25 @@ func EnhanceFromSchematic(ctx context.Context, prof profile.Profile, schematic *
 	}
 
 	// skip customizations for profile kinds which don't support it
-	if prof.Output.Kind != profile.OutKindInitramfs && prof.Output.Kind != profile.OutKindKernel && prof.Output.Kind != profile.OutKindInstaller {
+	//
+	// initramfs/kernel can't carry extra kernel args & META
+	// !secureboot (non-UKI) installer can't carry extra kernel args & META
+	// UKI installer has kernel args embedded in the UKI image
```

Comment:
if we want to support, we could later add as efi addons and sd-boot would pick them up